File: Pythonleetcode/Leetcode242.py
class Solution(object):
    def isAnagram(self, s, t):
        ta=[]
        tb=[]
        if len(s)!=len(t):
            return False
        for i in range(len(s)):
            ta.append(s[i])
            tb.append(t[i])
        if sorted(ta)==sorted(tb):
            return True
        else:return False




print(Solution().isAnagram( "ab", "bb"))

File: CppPractice/Project1/Project1/Leetcode242.cpp

#include <vector>

#include <iostream>

#include <stdlib.h>

#include <time.h>

#include <sstream>

#include <algorithm>

#include <map>

#include <bitset>

using namespace std;


class Solution {
public:
    bool isAnagram(string s, string t) {
        sort(s.begin(),s.end());
        sort(t.begin(), t.end());
        return s == t;

    }
};